.env.docker

```
@@ -34,5 +34,28 @@ DC_AUTH_CORS="http://localhost:5173,https://admin.medusa.localhost,http://localh
 DC_SENTRY_DSN=
 DC_SENTRY_NAME=
 
+# PPL Fulfillment Provider
+# Set to "1" to enable PPL service, "0" to disable
+DC_PPL_ENABLED=1
+DC_PPL_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DC_PPL_CLIENT_ID key should go before the DC_PPL_ENABLED key

(UnorderedKey)